manifests/params.pp

```diff
@@ -32,6 +32,10 @@
   $xpack                    = undef
   $systemd_override_dir     = '/etc/systemd/system/filebeat.service.d'
   $systemd_beat_log_opts_template = "${module_name}/systemd/logging.conf.erb"
+  $chocolatey_provider        = false
+  $chocolatey_package_name    = 'fileveat'
```

Comment:
I'm assuming this should be filebeat and is a typo?

Reply:
Yes indeed will fix it !